config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+      previousexpireddiscussion: "Per favore consulta prima <a href=\"%{storypath}\" \"target=_blank\">la discussione precedente</a> per questo articolo.  Se il contenuto è cambiato o giustifica una nuova discussione, puoi ripubblicarlo."
+      fetchtitle: "Recuperare il titolo"
+      url: "URL :"
+      title: "Titolo :"
+      changetitle: "Gli utenti hanno suggerito di modificare il titolo di questo articolo in:"
+      tags: "Etichette :"
+      changetags: "Gli utenti di hanno suggerito di modificare le etichette di questo articolo in:"
+      text: "Testo :"
+      placeholdertext: "Opzionale quando si pubblica un URL; per favore consulta le linee guida"
+      storysubmissionguidelines: "Linee guida per la pubblicazione degli articoli"
+      guidelinespart1: "Non modificare il titolo, se non quando il titolo originale dell'articolo manca di contestualizzazione o è confusionario. <strong>Per favore rimuovi dal titolo componenti estranei come il nome del sito o della sezione.</strong>"
+      guidelinespart2: "Quando l'articolo pubblicato è datato a più di un anno fa, per favore riporta l'anno di pubblicazione dell'articolo tra parentesi."
+      guidelinespart3: "Quando pubblichi un URL, la casella di testo è opzionale e va utilizzata solo per fornire contesto o spiegazioni qualora l'URL si riveli insufficiente. Le proprie interpretazioni o opinioni si riservino per un commento, in modo che questo possa essere valutato separatamente dall'articolo. "
+      guidelinespart4: "Se nessuna etichetta sembra essere adeguata per l'articolo che stai pubblicando, è probabile che non corrisponda agli <a href=\"/about\" \"target=_blank\">argomenti trattati</a>. Non strafare con le etichette che non sono temi principali dell'articolo."
+      guidelinespart5: "Per poter pubblicare facilmente una pagina che stai visualizzando nel tuo browser su gambe.ro, trascina questo bookmarklet sulla tua barra dei segnalibri:[<a href=\"javascript:{window.open(%22%{rooturl}stories/new?url=%22+encodeURIComponent(document.location)+%22&title=%22+encodeURIComponent(document.title));%20void(0);}\">Pubblicare su %{appname}</a>]. Sarete reindirizzati a questa pagina con l'URL e il titolo della pagina visualizzata."
+      submitter: "Il contributore è"
+      author: "Contributore :"
+      iamauthor: "Io sono"
+      authorownership: " il contributore di questo articolo a questo URL (o questo testo)"
+    listdetail:
+      storyremovedby: "[Articolo rimosso da %{user}]"
+      moderator: "il moderatore"
+      originalsubmitter: "il contributore originale"
+      authoredby: "pubblicato da"
+      via: "preso da"
+      justnow: "in questo istante"
+      edit: "modifica"
+      delete: "rimuovere"
+      confirmundelete: "Sei sicuro di voler ripristinare questo articolo"
+      undelete: "ripristinare"
+      confirmdelete: "Sei sicuro di voler rimuovere questo articolo?"
+      suggest: "suggerire"
```

Comment:
suggerisci?

Reply:
Nella traduzione francese il verbo "suggérer", che mi pare una forma dell'infinito; prima di confermare vorrei vederlo in uso.